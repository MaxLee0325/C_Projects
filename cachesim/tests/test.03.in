3
16
65536
L1 16 1 16
L2 32 2 16
L3 64 4 16
14
S 22 1234
L 22
S 48 3456
L 48
S 70 4567
L 70
S 4118 5678
L 4118
L 22
L 4118
S 2070 6789
L 2070
L 4118
L 22
