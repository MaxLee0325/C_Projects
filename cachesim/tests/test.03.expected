Level 1 cache: sets: 16, lines 1, block size: 16
Level 2 cache: sets: 32, lines 2, block size: 16
Level 3 cache: sets: 64, lines 4, block size: 16
Cache L3 miss @ 0x0016
Cache L2 miss @ 0x0016
Cache L1 miss @ 0x0016
Stored word: 0x4d2
Cache L1 hit @ 0x0016
Loaded word: 0x4d2
Cache L3 miss @ 0x0030
Cache L2 miss @ 0x0030
Cache L1 miss @ 0x0030
Stored word: 0xd80
Cache L1 hit @ 0x0030
Loaded word: 0xd80
Cache L3 miss @ 0x0046
Cache L2 miss @ 0x0046
Cache L1 miss @ 0x0046
Stored word: 0x11d7
Cache L1 hit @ 0x0046
Loaded word: 0x11d7
Cache L3 miss @ 0x1016
Cache L2 miss @ 0x1016
Cache L1 miss @ 0x1016
Stored word: 0x162e
Cache L1 hit @ 0x1016
Loaded word: 0x162e
Cache L2 hit @ 0x0016
Cache L1 miss @ 0x0016
Loaded word: 0x4d2
Cache L2 hit @ 0x1016
Cache L1 miss @ 0x1016
Loaded word: 0x162e
Cache L3 miss @ 0x0816
Cache L2 miss @ 0x0816
Cache L1 miss @ 0x0816
Stored word: 0x1a85
Cache L1 hit @ 0x0816
Loaded word: 0x1a85
Cache L2 hit @ 0x1016
Cache L1 miss @ 0x1016
Loaded word: 0x162e
Cache L3 hit @ 0x0016
Cache L2 miss @ 0x0016
Cache L1 miss @ 0x0016
Loaded word: 0x4d2
Cache: L1, hits: 5, misses: 9
Cache: L2, hits: 3, misses: 6
Cache: L3, hits: 1, misses: 5
